tensorbay/opendataset/CCPD/loader.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name, missing-module-docstring
+
+import os
+from typing import Dict, Iterator, Union
+
+from ...dataset import Data, Dataset
+from ...label import LabeledPolygon
+from .._utility.glob import glob
+
+_PROVINCES = (
+    "皖",
+    "沪",
+    "津",
+    "渝",
+    "冀",
+    "晋",
+    "蒙",
+    "辽",
+    "吉",
+    "黑",
+    "苏",
+    "浙",
+    "京",
+    "闽",
+    "赣",
+    "鲁",
+    "豫",
+    "鄂",
+    "湘",
+    "粤",
+    "桂",
+    "琼",
+    "川",
+    "贵",
+    "云",
+    "藏",
+    "陕",
+    "甘",
+    "青",
+    "宁",
+    "新",
+    "警",
+    "学",
+    "O",
```

Comment:
this `"O"` is suspicious?

Reply:
This's what the original author did. they use O as a sign of "no character" because there is no O in Chinese license plate characters.